### PR TITLE
Delete book

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**
Implemented deletion of books from database.

### CODE REVIEW

File: service/bookCrudService.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                cb(responseAttributes);
+        } else {
+            dao.delete(bookId, (error, result) => {
+                if (error) {
+                    responseAttributes.status = 500;
+                    responseAttributes.message =
+                        'There was an error while attempting to ' +
+                        'delete that book from the database.';
+                } else {
+                    responseAttributes.status = 200;
```
Comment (sean-darsie) [inline service/bookCrudService.js:70]: the status code for successfully deleting a book should be 204 no content.

**Review by sean-darsie — APPROVED** [review]
Approved with comment. Change the status code to 204 con content rather than 200

File: service/bookCrudService.js (hunk 1 above)
Comment (justin-obrien-smoothstack) [reply service/bookCrudService.js:70]: But I'm returning content here, a message telling the user that the book was successfully deleted.

Comment (sean-darsie) [reply service/bookCrudService.js:70]: ok. Sounds good.

**Pull request merged**